Sources/AppCoinsSDK/Domain/Entities/AppcSDKInternal.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 internal class AppcSDKInternal {
     
     static internal func initialize() {
-        MMPUseCases.shared.getAttribution()
+        MMPUseCases.shared.getAttribution { PayFlowUseCases.shared.setPayFlow() }
```

Comment:
We should not run attribution and payflow calls on the main thread, should be deprioritized to a background queue in order to give priority to the developer logic.